packages/p2p/src/components/modal-manager/modal-manager-context-provider.tsx

```diff
@@ -19,6 +19,10 @@ type TModalState = {
     stacked_modal: TModalVariants | null;
 };
 
+type TPersistedStates = {
+    [key: string]: any;
```

Comment:
i couldn't figure out exactly what type it was complaining when i put either type. best to leave this as any

Reply:
What about `Record<string, unknown>`

Reply:
let me check